src/test/java/com/splunk/hecclient/RawEventBatchTest.java

```diff
@@ -122,4 +127,31 @@ public void checkEquals() {
 
         Assert.assertFalse(batchOne.equals(batchTwo));
     }
+
+    @Test
+    public void testGZIPCompressionForRaw() {
```

Comment:
Can we add a test case where compression fails?

Reply:
It's already covered in this test case, so I don’t think to add a new test case